expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -81,11 +81,15 @@ private ExpandedPerson toExpandedPerson(String response, User owner) throws Json
 
     @Override
     public ExpandedDataEntry expandEntry(Entity dataEntry) throws JsonProcessingException, NotFoundException {
+        var identifier = dataEntry.getIdentifier();
         if (dataEntry instanceof Resource) {
+            logger.info("Expanding Resource: {}", identifier);
             return ExpandedResource.fromPublication(uriRetriever, dataEntry.toPublication(resourceService));
         } else if (dataEntry instanceof TicketEntry) {
+            logger.info("Expanding TicketEntry: {}", identifier);
             return ExpandedTicket.create((TicketEntry) dataEntry, resourceService, this, ticketService);
         } else if (dataEntry instanceof Message) {
+            logger.info("Expanding Message: {}", identifier);
```

Comment:
Hadde det vært verdifult for debuggings årsaker å logge hele json inputen hvis objektet hverken er Resource, TicketEntry eller Message?

Reply:
Det er trolig en god idé.

Reply:
Neste PR